diff -ru blender-2.55-beta.orig/intern/itasc/CMakeLists.txt blender-2.55-beta/intern/itasc/CMakeLists.txt
--- blender-2.55-beta.orig/intern/itasc/CMakeLists.txt	2010-11-17 00:35:21.000000000 +0100
+++ blender-2.55-beta/intern/itasc/CMakeLists.txt	2010-11-17 00:36:26.000000000 +0100
@@ -25,7 +25,7 @@
 # ***** END GPL LICENSE BLOCK *****
 
 SET(INC
-	../../extern/Eigen2
+	/usr/include/eigen2
 )
 
 SET(SRC
diff -ru blender-2.55-beta.orig/source/blender/ikplugin/CMakeLists.txt blender-2.55-beta/source/blender/ikplugin/CMakeLists.txt
--- blender-2.55-beta.orig/source/blender/ikplugin/CMakeLists.txt	2010-11-17 00:35:31.000000000 +0100
+++ blender-2.55-beta/source/blender/ikplugin/CMakeLists.txt	2010-11-17 00:38:05.000000000 +0100
@@ -41,7 +41,7 @@
 
 IF(WITH_IK_ITASC)
 	ADD_DEFINITIONS(-DWITH_IK_ITASC)
-	LIST(APPEND INC ../../../extern/Eigen2)
+	LIST(APPEND INC /usr/include/eigen2/)
 	LIST(APPEND INC ../../../intern/itasc)
 	LIST(APPEND SRC ./intern/itasc_plugin.cpp)
 ENDIF(WITH_IK_ITASC)
